ChildProject/pipelines/eafbuilder.py

```diff
@@ -51,6 +53,37 @@ def create_eaf(
         )
         eaf.add_annotation("context_" + eaf_type, context_onset, context_offset)
 
+    if speech_segments is not None:
+
+        from ChildProject.converters import AnnotationConverter
+
+        type_to_id = {
+            val: key
+            for key, val in reversed(
+                list(AnnotationConverter.SPEAKER_ID_TO_TYPE.items())
+            )
+        }
+
+        for segment in speech_segments.to_dict(orient="records"):
+            speaker_id = None
+
+            if "speaker_id" in segment:
+                speaker_id = segment["speaker_id"]
+            elif "speaker_type" in segment and segment["speaker_type"] in type_to_id:
+                speaker_id = type_to_id[segment["speaker_type"]]
+
+            if speaker_id is None:
+                continue
+
+            if speaker_id not in eaf.tiers:
+                eaf.add_tier(speaker_id)
```

Comment:
don't know if we should set other parameters of add_tier and if that case, which values should be assigned. e.g. what about ling? See https://github.com/dopefishh/pympi/blob/a41807e0b7d20fb014b95bc836a366933e7fe4a2/pympi/Elan.py#L375-L395

Reply:
At the moment, this has not been a problem, to keep in mind for future use